src/Tools/Md2Html/Program.cs

```diff
@@ -1,9 +1,16 @@
-﻿using System;
+using System;
 using System.IO;
-using NDesk.Options;
+using CommandLine;
+using CommandLine.Text;
 
 namespace Md2Html
 {
+    internal class CMDLineOptions
+    {
+        [Option('h', "help", Required = false, HelpText = "Show help and exit")]
```

Comment:
@pinzart90 ... two thoughts - I don't see that this PR updates any of the tests. I think that ndesk options and command line parser require setting flags differently... ie `/` vs `-` or `--`, I can't remember for certain.

Have you tried running the docs browser tests locally? Perhaps the markdown converter tool is never shutting down causing some kind of deadlock in the test runner etc.

Reply:
@mjkkirschner good catch, the md2html tests do seem to hang locally. Hopefully it is the same cause on the build machine